tests/framework/ROM/TimeSeries/SyntheticHistory/auto_arma.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries/SyntheticHistory.AutoARMA</name>
+    <author>talbpaul</author>
+    <created>2021-02-11</created>
+    <classesTested>SupervisedLearning.SyntheticHistory,TSA.Fourier,TSA.ARMA</classesTested>
+    <description>
+      Tests clustering for the SyntheticHistory ROM with Fourier and ARMA steps.
+    </description>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>AutoARMA</WorkingDir>
+    <Sequence>read, train, print, sample, plot</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Steps>
+    <IOStep name="read">
+      <Input class="Files" type="">infile</Input>
+      <Output class="DataObjects" type="HistorySet">indata</Output>
+    </IOStep>
+    <RomTrainer name="train">
+      <Input class="DataObjects" type="HistorySet">indata</Input>
+      <Output class="Models" type="ROM">synth</Output>
+    </RomTrainer>
+    <IOStep name="print">
+      <Input class="Models" type="ROM">synth</Input>
+      <Output class="DataObjects" type="DataSet">romMeta</Output>
+      <Output class="OutStreams" type="Print">romMeta</Output>
+    </IOStep>
+    <MultiRun name="sample">
+      <Input class="DataObjects" type="PointSet">placeholder</Input>
+      <Model class="Models" type="ROM">synth</Model>
+      <Sampler class="Samplers" type="MonteCarlo">mc</Sampler>
+      <Output class="DataObjects" type="HistorySet">samples</Output>
+      <Output class="OutStreams" type="Print">samples</Output>
+    </MultiRun>
+    <IOStep name="plot">
+      <Input class="DataObjects" type="HistorySet">indata</Input>
+      <Input class="DataObjects" type="HistorySet">samples</Input>
+      <Output class="OutStreams" type="Plot">compare_0</Output>
+      <Output class="OutStreams" type="Plot">compare_1</Output>
+    </IOStep>
+  </Steps>
+
+  <Files>
+    <Input name="infile">../TrainingData/GlobalFourierARMA_A.csv</Input>
+  </Files>
+
+  <Samplers>
+    <MonteCarlo name="mc">
+      <samplerInit>
+        <limit>2</limit>
+        <initialSeed>42</initialSeed>
+      </samplerInit>
+      <constant name="scaling">1.0</constant>
+    </MonteCarlo>
+  </Samplers>
+
+  <Models>
+    <ROM name="synth" subType="SyntheticHistory">
+      <Target>signal0, signal1, pivot</Target>
+      <Features>scaling</Features>
+      <pivotParameter>pivot</pivotParameter>
+      <fourier target="signal0,signal1">
+        <periods>2, 5, 10, 20, 50</periods>
+      </fourier>
+      <gaussianize target="signal0, signal1"/>
+      <autoarma target="signal0, signal1" seed='42'>
+        <P_upper>5</P_upper>
+        <Q_upper>5</Q_upper>
+        <criterion>bic</criterion>
+      </autoarma>
+      <arma target="signal0, signal1" auto_select="True" seed='42'>
+        <P>1</P> <!-- NOTE: because `auto-select` is True, these are ignored -->
+        <Q>0</Q> <!-- NOTE: because `auto-select` is True, these are ignored -->
+      </arma>
```

Comment:
I suggest instead of ignoring the user-provided P and Q, if auto_select = "True", either to error out or to consider them as P_upper, or lower, and the same for Q.

Reply:
P_upper and Q_upper are already used in the `autoarma` algorithm, so I think maybe I'll just have it error out if the user provides P and Q here when `auto-select` is True